app/build.gradle

```diff
@@ -288,6 +287,9 @@ dependencies {
   implementation "com.simplecityapps:recyclerview-fastscroll:$versions.fastScroll"
   implementation "com.google.android:flexbox:$versions.flexbox"
   implementation "com.mixpanel.android:mixpanel-android:$versions.mixpanel"
+
+  implementation "androidx.work:work-runtime:$versions.workManager"
+  implementation "androidx.work:work-gcm:$versions.workManager"
```

Comment:
Curious,

1. Why are we removing the KTX version.
2. Why a GCM artifact?

Reply:
1. [Work Runtime KTX](https://developer.android.com/kotlin/ktx#workmanager) only adds support for coroutines, which we do not use at the moment. I felt we could use the core library instead.

2. For the GCM artifact, please see the release notes linked here: https://developer.android.com/jetpack/androidx/releases/work#2.2.0